### PR TITLE
799 mr bmi 

### PR DESCRIPTION
**Pull request by RamilGaripov**
Ticket: metriport/metriport-internal#799

### Dependencies

- Upstream: none
- Downstream: none

### Description

Create BMI Mr Summary behind FF.

### Testing

- Local
  - [x] Genrates MR valid BMI summary 
- Staging
  - [x] Genrates MR valid BMI summary for specified cx
  - [x] Does not generate BMI summary for unspecified cx
- Production
  - [ ] Genrates MR valid BMI summary for specified cx
  - [ ] Does not generate BMI summary for unspecified cx
 
### Release Plan

- [x] Add FF to appconfig
- [ ] Merge this

### CODE REVIEW

**Review by Orta21 — APPROVED** [review]

File: packages/core/src/external/aws/lambda-logic/bundle-to-html-bmi.ts [hunk 1]
```diff
@@ -311,21 +315,21 @@ export function bundleToHtml(fhirBundle: Bundle, brief?: Brief): string {
...
                   }
                 },
-                y: { 
-                  title: { 
-                    display: true, 
-                    text: 'BMI Value' 
+                y: {
+                  title: {
+                    display: true,
+                    text: 'BMI Value'
```
Comment (RamilGaripov) [inline packages/core/src/external/aws/lambda-logic/bundle-to-html-bmi.ts:329]: Missed an update?

File: packages/core/src/external/aws/lambda-logic/bundle-to-html-bmi.ts [hunk 2]
```diff
@@ -636,21 +642,88 @@ function createBmiFromObservationVitalsSection(observations: Observation[]): {
...
+} {
+  if (!observations) {
+    return { hba1cSection: "", hba1cChartData: { labels: [], data: [] } };
+  }
+
+  const a1cLoincCode = "4548-4";
+
+  const hba1cObservations = observations.filter(observation => {
+    const observationDisplay = observation.code?.coding?.find(coding => {
+      return coding.code === a1cLoincCode;
```
Comment (RamilGaripov) [inline packages/core/src/external/aws/lambda-logic/bundle-to-html-bmi.ts:683]: nit: could be worth to use `trim()` prior to comparison

File: packages/core/src/external/aws/lambda-logic/bundle-to-html-bmi.ts [hunk 3]
```diff
@@ -1207,7 +1280,7 @@ function createMedicationSection(
 
   const recentMedications = medicationStatements.filter(medicationStatement => {
     return (
-      dayjs(medicationStatement.effectivePeriod?.start).isAfter(dayjs().subtract(3, "year")) ||
+      dayjs(medicationStatement.effectivePeriod?.start).isAfter(dayjs().subtract(4, "months")) ||
```
Comment (RamilGaripov) [inline packages/core/src/external/aws/lambda-logic/bundle-to-html-bmi.ts:1283]: just a thought:

this seems oddly specific. Do we want to add a note somewhere in the MR section, like `Medications (last 4 months)`

Comment (Orta21) [reply packages/core/src/external/aws/lambda-logic/bundle-to-html-bmi.ts:1283]: I mean this is something they specifically wanted - once we add the MR summary engine it wont be an issue

**Pull request merged**